volim.html

```diff
@@ -116,5 +85,28 @@
       </div>
    </div>
   </div>
+  <div class="contact">
+    <div class="contact-white">
+      <div class="page-title">
+        <p class="page-title__text">contact us</p>
+        <div class="page-title__circle"></div>
+      </div>
+    </div>
+    <div class="contact-red">
+      <div class="contact-form">
+        <input  class="contact-form__input" type="text" name="name" placeholder="Your Name">
+        <input  class="contact-form__input" type="text" name="phone-number" placeholder="Phone Number">
+        <input  class="contact-form__input" type="email" name="email" placeholder="Email">
+        <textarea class="contact-form__textarea" type="text" name="message" placeholder="Message"></textarea>
+        <button class="contact-form__button">Send</button>
+      </div>
+      <div class="contact-img">
+        <img class="contact-img__map"c src="img/map_img.jpg">
+      </div>
+    </div>
+  </div>
+  <footer>
```

Comment:
not fixed :)

Reply:
![image](https://user-images.githubusercontent.com/61840948/104616681-443d9580-5693-11eb-96cc-458029584e8c.png)